src/lseq.rs

```diff
@@ -114,77 +112,55 @@ pub enum Op<T, A: Actor> {
     },
 }
 
-impl<T, A: Actor> LSeq<T, A> {
-    /// Create an LSEQ for the empty string
+impl<T: Clone, A: Actor> LSeq<T, A> {
```

Comment:
You can't do anything useful without `T: Clone` so don't even let people instantiate an LSeq if T is not Clone.